### ISSUE TITLE
urlparse result does not raise AttributeError for unsupported attributes

### ISSUE THREAD

**Issue by lopuhin** [scrapy/scurl#45]
```
>>> import scurl
>>> x = scurl.urlparse('http://example.com/bar')
>>> x.fooo is None
True
```
while an ``AttributeError`` should be raised. This might be fixed by #42 though.

**Comment by malloxpb** [comment]
Hey @lopuhin , I believe this has been fixed :)

**Comment by lopuhin** [comment]
Right, thanks for the fix @nctl144 ! (fixed in #55 )
